executable-spec/src/Cardano/Ledger/Spec/STS/Update/Data.hs

```diff
@@ -126,10 +181,15 @@ newtype URL = URL { getText :: Text }
   deriving stock (Eq, Ord, Show, Generic)
   deriving newtype (ToCBOR)
 
+-- | Hash of the SIP contents (`SIPData`) also plays the role of a SIP
+-- unique id
+data SIPHash hashAlgo = SIPHash (Hash hashAlgo SIPData)
```

Comment:
Why not simply using `Hash hashAlgo SIPData`?

Reply:
You mean as a type synonym?